spec/unit/knife/bootstrap_spec.rb

```diff
@@ -1,6 +1,6 @@
 #
 # Author:: Ian Meyer (<ianmmeyer@gmail.com>)
-# Copyright:: Copyright 2010-2016, Ian Meyer
+# Copyright:: Copyright 2010-2019, Chef Software Inc.
```

Comment:
We need to leave the original copyright here. We can add another line below it.

Reply:
Done. Thanks